esp/bindings/SOAP/Platform/soapservice.cpp

```diff
@@ -129,6 +129,34 @@ int CSoapService::processHeader(CHeader* header, IEspContext* ctx)
     return returnValue;
 }
 
+void CSoapService::addMessageToTraceSummary(IEspContext* ctx, CSoapValue* soapValue, const char* parent)
```

Comment:
I'm not sure we should add every parameter from the request like this.  I think what parameters are txsummarized should probably be method specific.  This could be huge and hard to read.

Even if we did I think blank values shouldn't even have the parameter name included.

If users want the request itself logged there are already config options for that.